### DISCUSSION TITLE
Timelapse

### DISCUSSION THREAD

**Discussion by weedy171** [signag/raspi-cam-srv#21] [General]
Love the project. Is their a way to specify sun rise and sunset. I like to use this project to setup a PI camera for a small 3-4 month project and only record during daylight.
Thanks!

**Comment by signag** [comment]
Thanks for reminding me.
Actually, a similar project was the reason why I started this.
But then there were so many sidesteps.
I'll come back to that.

**Comment by signag** [comment]
The feature is now available in V2.8.0
Hope it fits your needs.
